apps/accounts/automations/change_secret/manager.py

```diff
@@ -180,7 +180,7 @@ def on_host_success(self, host, result):
         while retry_count < max_retries:
             try:
                 recorder.save()
-                account.save(update_fields=['secret', 'version', 'date_updated'])
+                account.save(update_fields=['secret', 'date_updated'])
                 break
             except Exception as e:
                 retry_count += 1
```

Comment:
There is no known issue with this code snippet related to the provided context. The function appears to be correctly implemented without any irregularities or potential issues according to current knowledge cutoff, but it does seem that there could be improvement options:

1) Consider using `logging` instead of print statements for debugging purposes to have more structured log outputs.

2) Since only one field (`update_fields`) needs modification, consider using `get()` method along with `list()`, so you don't change what's already stored.

3) Ensure to test and modify based on actual changes in requirements/dependencies if needed.
  
No other suggestions can be made without specific feedback or context from your project specification.